pkg/reconciler/containersource/containersource_test.go

```diff
@@ -637,7 +637,7 @@ func TestObjectHasDeleteTimestamp(t *testing.T) {
 	if diff := cmp.Diff(want, got); diff != "" {
 		t.Errorf("unexpected returned object (-want, +got) = %v", diff)
 	}
-	var wantErr error = nil
+	var wantErr error
 	if diff := cmp.Diff(wantErr, gotErr); diff != "" {
```

Comment:
Why does this not just check that `gotErr == nil`?

Reply:
Done